### ISSUE TITLE
Definition

### ISSUE THREAD

**Issue by bphd** [michael-act/Senginta#6]
Is your repo meant to create a software to search on many search engines at the same time (in one search box)?

If yes, where is the difficulty to integrate more than the two actually implemented search engines?

**Comment by michaelact** [comment]
Nope. But, sorry in advance for not responding quickly. Senginta is a Software that includes Google Books, Google Search, Google Shop, etc to run independently. Please kindly check `How to use: Video` in README.md to find out more about how Senginta works.
